pubobot/bot.py

```diff
@@ -5,10 +5,10 @@
 import datetime
 import re
 import random
-from discord import errors
+import discord
 from itertools import combinations
-from collections import OrderedDict
-from . import client, config, console, stats3, scheduler, utils
+from . import client, config, console, memberformatter, stats3, scheduler, utils
+from typing import List, OrderedDict
```

Comment:
I would do two things here:

1. Place local imports after imports from the standard library and third party imports. Mixing them in leads to confusion.
2. Import OrderedDict from `collections`, the one in `typing` works but it's an alias and was deprecated in 3.9.

Reply:
> I would do two things here:
> 
>     1. Place local imports after imports from the standard library and third party imports. Mixing them in leads to confusion.
> 
>     2. Import OrderedDict from `collections`, the one in `typing` works but it's an alias and was deprecated in 3.9.

2. same w/ List?

Reply:
oh nvm List isn't part of collections